indigo_app/templates/indigo_api/work_tasks.html

```diff
@@ -0,0 +1,37 @@
+{% extends "indigo_api/work_layout.html" %}
+
+{% block title %}Tasks – {{block.super }}{% endblock %}
+
+{% block work-content %}
+  <div class="container mt-3">
```

Comment:
I don't think you need this container. It's doubled up from the parent template.